src/styles/components/_card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: 20rem;
+  margin: auto;
+  padding: 3.2rem 1.6rem 1.6rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @include border(0.1rem, $color-card-bgc, 0.5rem);
```

Comment:
It seems you're using an `@include` for border here. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), if you are using SCSS, you should avoid using `@include` for common CSS properties where it's not necessary. Please use the standard `border` property instead.